core/src/sql/statements/define/table.rs

```diff
@@ -37,8 +38,24 @@ pub struct DefineTableStatement {
 	pub if_not_exists: bool,
 	#[revision(start = 3)]
 	pub kind: TableType,
+	/// Should we overwrite the field definition if it already exists
 	#[revision(start = 4)]
 	pub overwrite: bool,
+	/// The last time that a DEFINE FIELD was added to this table
+	#[revision(start = 5)]
+	pub cache_fields_ts: Uuid,
+	/// The last time that a DEFINE EVENT was added to this table
+	#[revision(start = 5)]
+	pub cache_events_ts: Uuid,
+	/// The last time that a DEFINE TABLE was added to this table
+	#[revision(start = 5)]
+	pub cache_tables_ts: Uuid,
+	/// The last time that a DEFINE INDEX was added to this table
+	#[revision(start = 5)]
+	pub cache_indexes_ts: Uuid,
+	/// The last time that a LIVE query was added to this table
+	#[revision(start = 5)]
+	pub cache_lives_ts: Uuid,
```

Comment:
As we're relying on transaction conflict management to prevent concurrent schema updates across SurrealDB nodes, would it be feasible to replace UUIDv7 with a simpler u64 sequence that increments on each update? This could reduce complexity since we only need uniqueness within the scope of sequential updates.
This aligned with what we currently do on stored BTree nodes
https://github.com/surrealdb/surrealdb/blob/9927fc654a35ced8c90a60bb6c32f02fef777de4/core/src/idx/trees/btree.rs#L39

Reply:
I guess the question is, are people ever going to make 18446744073709551615 schema changes or live queries hehehe